plugins/k8saudit-eks/pkg/k8sauditeks/k8sauditeks.go

```diff
@@ -137,6 +138,10 @@ func (p *Plugin) Open(clustername string) (source.Instance, error) {
 		for {
 			select {
 			case i := <-eventsC:
+				message := *i.Message
+				if strings.Contains(message, "[Truncated...]") {
```

Comment:
Is there a more performant way of doing this check? Like, if we know for sure that `[Truncated...]` is at the end of the string, could `strings.HasSuffix` be more efficient?

Also, is there anything better than we can do instead of just skipping? e.g. are there any frameworks out there for broken JSON repairing or something? In the worst case scenario, can we at least log that we are discarding a meaningful event?

Reply:
Ok for the `strings.HasSuffix`, I'll use it.

I searched for a way to fix the broken JSON, I didn't find anything relevant, too much brackets and braces to fix. I can continue to search anyway.
I'll add a log line for now.

Reply:
I added a log line with the auditID of the truncated line

Reply:
@Issif 

Should we switch to `strings.HasSuffix` or stick with `strings.Contains` ? :thinking:

Reply:
I switched to `strings.HasSuffix`